src/PVRIptvData.cpp

```diff
@@ -802,7 +802,21 @@ bool PVRIptvData::LoadPlayList(void)
       channel.iTvgShift         = tmpChannel.iTvgShift;
       channel.bRadio            = tmpChannel.bRadio;
       channel.properties        = tmpChannel.properties;
-      channel.strStreamURL      = strLine;
+      
+      auto propPair = channel.properties.find("http-user-agent");
+      if (propPair != channel.properties.end())
+      {
+        channel.strStreamURL = AddHeaderToStreamUrl(strLine, "user-agent", propPair->second);
+      }
+      else if (!g_userAgent.empty())
+      {
+        channel.strStreamURL = AddHeaderToStreamUrl(strLine, "user-agent", g_userAgent);
+      }
+	  else
```

Comment:
Indentation is wrong. Should be inline with rest of statement.

Reply:
Yeah, noticed.  Now it's correct.